connector/exceptionsconnector/README.md

```diff
@@ -0,0 +1,75 @@
+# Exceptions Connector
+
+<!-- status autogenerated section -->
+| Status                   |               |
+| ------------------------ |---------------|
+| Stability                | [alpha] |
+| Supported pipeline types | See [Supported Pipeline Types](#supported-pipeline-types)  |
+| Distributions            | [contrib]     |
+
+## Supported Pipeline Types
+
+| [Exporter Pipeline Type] | [Receiver Pipeline Type] |
+| ------------------------ | ------------------------ |
+| traces                   | metrics, logs            |
+
+Extract metrics and logs from [recorded exceptions](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/exceptions/) from spans.
+<!-- end autogenerated section -->
+
+## Overview
+
+Each metric will have _at least_ the following dimensions because they are common across all spans:
+- Service name
+- Span kind
+- Status code
+- Exception message
+- Exception type
```

Comment:
Shouldn't it include span.id and trace.id for logs? And exemplars for metrics?

Reply:
Also considering for a follow up PR, wdyt?

Reply:
Yes, as long as we don't have a release of the contrib binary with this component before we have that in place.

Reply:
Could you please create a tracking issue and mark this as resolved?